### PR TITLE
Total contributors

### PR DESCRIPTION
**Pull request by amunz**

### CODE REVIEW

File: activity-metrics/total-contributors.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## 2. Use Cases
+
+## 3. Sample Visualization
+### Kibble - GitHub
+![Total Contributors visual](https://user-images.githubusercontent.com/22136995/38274804-6583433a-3755-11e8-9871-0de4c300a2ff.png)
+
+## 4. Sample Implementations
+### Kibble - GitHub
+First, it grabs all the committers for one or multiple projects.
+```
```
Comment (GeorgLink) [inline activity-metrics/total-contributors.md]: Indicate programming language (eg. py for Python) to get syntax highlighting.

File: activity-metrics/total-contributors.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Total Contributors 
+
+## 1. Description
+The total number of contributors over time on any platform.
+
+## 2. Use Cases
+
+## 3. Sample Visualization
```
Comment (GeorgLink) [inline activity-metrics/total-contributors.md]: Please use the updated template that includes "Sample Filters"

**Comment by amunz** [comment]
Made the requested changes :)

**Comment by GeorgLink** [comment]
Merging with lazy consensus achieved.

**Comment by GeorgLink** [comment]
Thanks @amunz

**Pull request merged**